### PR TITLE
In some cases if and else match the same conditions. I fix it.

### PR DESCRIPTION
**Pull request by mosfet80**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [ ]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [ ]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [ ]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [ ]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [ ]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the issue ID number from our [Issues database](https://github.com/FreeCAD/FreeCAD/issues) in case a particular commit solves or is related to an existing issue. Ex: `Draft: fix typos - fixes #4805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.20 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=56135).

---

### CODE REVIEW

File: cMake/FindPyCXX.cmake [hunk 1]
```diff
@@ -27,17 +27,13 @@ if(PYCXX_INCLUDE_DIR)
     # headers better be in there
     if(NOT EXISTS "${PYCXX_INCLUDE_DIR}/CXX/Config.hxx")
         if(PyCXX_FIND_REQUIRED)
-            MESSAGE(FATAL_ERROR
-                "PyCXX: could not find CXX/Config.hxx in PYCXX_INCLUDE_DIR "
-            "${PYCXX_INCLUDE_DIR}")
+            MESSAGE(FATAL_ERROR "PyCXX: could not find CXX/Config.hxx in PYCXX_INCLUDE_DIR "  "${PYCXX_INCLUDE_DIR}")
```
Comment (luzpaz) [inline cMake/FindPyCXX.cmake:30]: Did you leave 2 whitespaces in front of `"${PYCXX_INCLUDE_DIR}")` for a reason?

File: cMake/FindPyCXX.cmake [hunk 2]
```diff
@@ -27,17 +27,13 @@ if(PYCXX_INCLUDE_DIR)
...
         else(PyCXX_FIND_REQUIRED)
-            MESSAGE(WARNING
-                "PyCXX: could not find CXX/Config.hxx in PYCXX_INCLUDE_DIR "
-            "${PYCXX_INCLUDE_DIR}")
+            MESSAGE(WARNING "PyCXX: could not find CXX/Config.hxx in PYCXX_INCLUDE_DIR "  "${PYCXX_INCLUDE_DIR}")
```
Comment (luzpaz) [inline cMake/FindPyCXX.cmake:32]: Did you leave 2 whitespaces in front of `"${PYCXX_INCLUDE_DIR}")` for a reason?

**Review by luzpaz — CHANGES_REQUESTED** [review]

**Comment by chennes** [comment]
I believe this is a misunderstanding of cMake -- from the cMake documentation:

> Per legacy, the [else()](https://cmake.org/cmake/help/latest/command/else.html#command:else) and [endif()](https://cmake.org/cmake/help/latest/command/endif.html#command:endif) commands admit an optional <condition> argument. **If used, it must be a verbatim repeat of the argument of the opening if command.**

(Emphasis mine)

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6654">pipeline status</a> for feature branch [PR_6654](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6654). Pipeline [500589299](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/500589299) was triggered at [8506b31](https://github.com/FreeCAD/FreeCAD/pull/6654/commits/8506b3159d5cd5eedd940cdae7bba1c32ff8288c). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Pull request closed without merging**